templates/virtual-baremetal-servers.yaml

```diff
@@ -67,8 +67,6 @@ resources:
         - - {get_param: bmc_prefix}
           - {get_param: suffix}
       network: {get_param: private_net}
-      security_groups:
```

Comment:
I would have thought that the security groups have essentially no effect because we run the noop firewall driver.  Given that, it seems fine to remove them, but I'm surprised they would have ever caused problems.